apps/report-documents-not-received/views/accessibility.html

```diff
@@ -0,0 +1,55 @@
+{{<layout}}
+
+  {{$header}}
+    {{header}}
+  {{/header}}
+
+  {{$content}}
```

Comment:
Im not sure if we need all these fields. I reckon if we use 

```
{{<partials-page}}
  {{$page-content}}
```

That should inherit the title ?